kivy/tests/test_animations.py

```diff
@@ -52,22 +59,22 @@ def sleep(t):
     from time import time, sleep
     from kivy.clock import Clock
     tick = Clock.tick
-    start = time()
```

Comment:
I think we need to put a `tick()` before any call to `time`. This may perhaps be the cause of the bug.